static/modules/collision.js

```diff
@@ -109,6 +109,19 @@ export function isColliding(entityA, entityB) {
   return false;
 }
 
+/**
+ * Determines if two entities are colliding and if damage should take place
+ * @param {Entity} entityA
+ * @param {Entity} entityB
+ * @param {number} cheatRadius
+ */
+export function isCollidingCheat(entityA, entityB, cheatRadius) {
```

Comment:
"Cheat" is a weird name... If i don't misunderstand it's meaning, wouldn't something better be "withinCollisionCheckRange" or something along those lines?